app-drawer/app-drawer.html

```diff
@@ -91,9 +91,10 @@
         width: var(--app-drawer-width, 256px);
         padding: 120px 0;
 
-        transition: 0.2s ease;
         transition-property: -webkit-transform;
         transition-property: transform;
+        transition-duration: inherit;
+        transition-timing-function: inherit;
```

Comment:
yeah, it doesn't work in Safari, but it does work in Chrome and Firefox. http://jsbin.com/fifewi/edit?html,console.